README.md

```diff
@@ -1,3 +1,27 @@
+# OMI Python Script Provider
+
+## How to build
+
+### Dependencies to build OMI Python Script Provider:
+- Install gcc and g++
+  - Redhat/CentOS: `sudo yum install gcc gcc-c++`
+  - Debian/Ubuntu OS: `sudo apt-get install gcc g++`

```

Comment:
I don't know which gcc version should be suitable for python provider, I was thinking that we should use latest version.